### PR TITLE
rollback model on transition

### PR DESCRIPTION
**Pull request by nbittich**

### CODE REVIEW

File: app/routes/measure-concepts/new.js [hunk 1]
```diff
@@ -11,4 +11,10 @@ export default class MeasureConceptsNewRoute extends Route {
       roadSignConcepts: this.store.findAll('road-sign-concept'),
     });
   }
+  resetController(controller) {
+    const measureConcept = controller.model.newMeasureConcept;
+    measureConcept.rollbackAttributes();
+    measureConcept.belongsTo('roadSignConcept').reload();
```
Comment (Windvis) [inline app/routes/measure-concepts/new.js:17]: Can you double check if this is needed? I thought new records would be deleted when you use `.rollbackAttributes()`. Can it even do a reload of a relationship that hasn't been saved yet?

Comment (nbittich) [reply app/routes/measure-concepts/new.js:17]: indeed, it wasn't needed, I removed them and will push in a second commit for clarity

Comment (Windvis) [reply app/routes/measure-concepts/new.js]: I'm not 100% sure but I think the rollbackAttributes on the new pages should have been kept. AFAIK now there will be "dangling" records in the store. (so if you enter the new route 5 times without saving, there will be 5 "new" records in the store.

Comment (nbittich) [reply app/routes/measure-concepts/new.js]: I checked it and it doesn't seem to dangle. can you join back the google meet so I can show you?

**Pull request merged**